igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/client/logparser/PropertyParser.java

```diff
@@ -29,7 +29,7 @@
 public class PropertyParser {
 
   private static final Logger log = LoggerFactory.getLogger(PropertyParser.class);
-  private static final String MAGIC_SEARCH_STRING = "SPINNAKER_PROPERTY_";
+  private static final String MAGIC_SEARCH_STRING = ".*SPINNAKER_PROPERTY_";
```

Comment:
What's the intent behind this change?

Reply:
Gitlab logs include the ANSI color codes, without this you can end up with garbled properties like `[32;1mMY_PROPERTY=something`, which in turn was causing invalid XML outputs.  The wildcard just ensures everything before and included SPINNAKER_PROPERTY gets removed from the key name.

Reply:
Is there no way to get gitlab to stop using the color codes?  This feels like it's gonna get mis-used somehow, or be the cause of bugs, like if someone has FOO_SPINNAKER_PROPERTY_ and SPINNAKER_PROPERTY, or something like that.

Reply:
There isn't any way I can see to change Gitlab's output but I did dig into this more and I think I have a better fix.  It's actually kind of bugged in igor/master at the moment, and that was the root cause of my issues.  I have a PR on my fork I can merge if you prefer that approach over this one [here](https://github.com/linjmeyer/igor/pull/2).  The Gitlab logs include the bash commands and their output (plus color codes) and the current setup is matching any line with the key, not just if its the start of the line only.  I fixed that (which fixes my issue) and also ensures `FOO_SPINNAKER_PROPERTY_ ` can't be used.  Check out the PR and let me know, thanks!

Reply:
I do like https://github.com/linjmeyer/igor/pull/2 !  Thank you!